tests/db/utils_test.py

```diff
@@ -5,6 +5,8 @@
 from __future__ import absolute_import
 
 import email
+import os
+import os.path
```

Comment:
It's the way that `os.path` is implemented. It has something to the effect of:
```python
if platform == 'Windows':
     import ntpath as path
elif platform == 'Posix':
     import posixpath as path
...
```

Since that has been the behavior for a **long** time, I very much doubt it will change before python 4.